go.mod

```diff
@@ -144,7 +144,7 @@ replace (
 	github.com/gin-gonic/gin => github.com/gin-gonic/gin v1.7.0
 	github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alpha.regen.1
 
-	github.com/tendermint/tendermint => github.com/sei-protocol/sei-tendermint v0.1.147
+	github.com/tendermint/tendermint => github.com/sei-protocol/sei-tendermint v0.1.129
```

Comment:
ignore, just wanted to confirm the data race issue is related to a tendermint change. Which seems to be the case :(